docs/engineering/technical-planning.md

```diff
@@ -0,0 +1,36 @@
+# Technical Planning Docs
+## Request for Comment
+In the course of planning development, there are often scenarios where a technology or implementation decision fork is presented and an engineer wants to solicit ideas or feedback. 
+
+Writing a full-blown Architectural Decision Record may not appropriate as that document requires significant overhead in proposing answers that still may be largely unknown. In most cases, we only want to invest the time in writing an ADR only when a decision has been made and needs to be codified. Until then, it's a Decision Proposal, not a Decision Record.
+
+Instead we use a Request for Comment markdown template that is more lightweight and speculative. It is designed to generate conversation and rapid feedback when there is not a clear solution with any level of certainty.
```

Comment:
>Instead we use a Request for Comment markdown template that is more lightweight and speculative. It is designed to generate conversation and rapid feedback when there is not a clear solution with any level of certainty.

"Instead we use a ..." seems a tad ambiguous and disjointed from the previous paragraph. Offering some clarification:

There are cases where there is not a clear solution with any level of certainty and we desire to generate conversation and rapid feedback. In these situations we use a Request for Comment <b>(RFC)</b> markdown template and follow the workflow described below to generate said feedback and conversation. An RFC by nature is designed to be more lightweight and speculative than a full ADR or ADP. RFC outcomes are described below.

Reply:
thanks